pkg/envoy/lds/egress.go

```diff
@@ -17,8 +17,11 @@ import (
 )
 
 const (
+	// @prometheus-metric
 	egressHTTPFilterChainPrefix = "egress-http"
-	egressTCPFilterChainPrefix  = "egress-tcp"
+
+	// @prometheus-metric
+	egressTCPFilterChainPrefix = "egress-tcp"
```

Comment:
Are these prefixes used to scrape the egress metrics?

Reply:
I don't believe so. I must have misunderstood your earlier message.

Reply:
Let's remove it in that case.